packages/flutter/lib/src/material/button.dart

```diff
@@ -58,9 +59,9 @@ class RawMaterialButton extends StatefulWidget {
   /// If this is set to null, the button will be disabled, see [enabled].
   final VoidCallback onPressed;
 
-  /// Padding to increase the size of the gesture detector which doesn't
-  /// increase the visible material of the button.
-  final EdgeInsets outerPadding;
+  /// Constraints used to increase the gesture detector size without increasing
+  /// the visible material of the button.
+  final BoxConstraints outerConstraints;
```

Comment:
I guess we should mention here that if this changes from null to non-null, any ongoing splashes, and any state in the `child`, will be lost.

Reply:
(so e.g. to animate this, we should animate from zero to a non-zero value, not null to a non-zero value)

Reply:
In this case, would it be better to default to an minWidth 0 and minHeight 0 everywhere?